plinth/modules/minetest/templates/minetest.html

```diff
@@ -0,0 +1,65 @@
+{% extends "service.html" %}
+{% comment %}
+    #
+    # This file is part of Plinth.
+    #
+    # This program is free software: you can redistribute it and/or modify
+    # it under the terms of the GNU Affero General Public License as
+    # published by the Free Software Foundation, either version 3 of the
+    # License, or (at your option) any later version.
+    #
+    # This program is distributed in the hope that it will be useful,
+    # but WITHOUT ANY WARRANTY; without even the implied warranty of
+    # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    # GNU Affero General Public License for more details.
+    #
+    # You should have received a copy of the GNU Affero General Public License
+    # along with this program.  If not,see
+    <http://www.gnu.org/licenses/>.
+    #
+{% endcomment %} 
+
+{% load bootstrap %}
+{% load i18n %}
+{% block status %}
+  {% if show_status_block %}
+    <h3>{% trans "Status" %}</h3>
+    <p class="running-status-parent">
+      {% with service_name=service.name %}
+        {% if service.is_running %}
+          <span class="running-status active"></span>
+          {% blocktrans trimmed %}
+            Service <em>{{ service_name }}</em> is running.
+          {% endblocktrans %}
+        {% else %}
+          <span class="running-status inactive"></span>
+          {% blocktrans trimmed %}
+            Service <em>{{ service_name }}</em> is not running.
+          {% endblocktrans %}
+        {% endif %}
+      {% endwith %}
+    </p>
+  {% endif %}
+  <div class="row">
+    <div class="col-sm-6">
+      <table class="table table-bordered table-condensed table-striped">
+        <thead>
+        <tr>
+          <th>{% trans "Address" %}</th>
+          <th>{% trans "Port" %}</th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr>
+          <td>
+            {% for domain in domains %}
```

Comment:
When `domains` is an empty list, this will cause one row to be created with an empty Address field and 30000 in the Port field.
Put the `for` loop around the `<tr>` elements instead, so that no rows will be shown in case of an empty list.